platform/raspberry-pi/platform_pi.c

```diff
@@ -782,7 +782,62 @@ INT wifi_getApInterworkingServiceEnable(INT apIndex, BOOL *output_bool)
 
 INT wifi_sendActionFrame(INT apIndex, mac_address_t MacAddr, UINT frequency, UCHAR *frame, UINT len)
 {
-    return 0;
+    /*
```

Comment:
Instead of fully implementing, please call wifi_hal_send_mgmt_frame in this function as you have suggested in the conversation.
i.e.
 return wifi_hal_send_mgmt_frame(apIndex, MacAddr, frame, len, frequency);

Reply:
Thanks, I appreciate the clarification.